decidim-proposals/config/locales/en.yml

```diff
@@ -94,6 +94,13 @@ en:
           email_outro: You have received this notification because you are following %{author_nickname}. You can stop receiving notifications following the previous link.
           email_subject: New proposal by %{author_nickname}
           notification_title: The <a href="%{resource_path}">%{resource_title}</a> proposal was created by <a href="%{author_path}">%{author_name} %{author_nickname}</a>.
+        proposal_endorsed_event:
+          email_intro: |-
+            Hi,
```

Comment:
We shouldn't add "Hi, newline" since the email already has a greeting. (It was wrongly used in some other events)